scripts/azureml-assets/azureml/assets/config.py

```diff
@@ -5,7 +5,20 @@
 from enum import Enum
 from pathlib import Path
 from typing import Dict, List
-from yaml import safe_load
+from yaml import safe_load, dump
+from subprocess import PIPE, run, STDOUT
+from util import logger
+import azureml.evaluate.mlflow as mlflow
+from mlflow.models import ModelSignature
+from sqlalchemy import true
+from transformers import  AutoTokenizer, AutoConfig, pipeline
+from transformers import (
+    AutoModelForSequenceClassification,
+    AutoModelForMaskedLM,
+    AutoModelForTokenClassification,
+    AutoModelForQuestionAnswering,
+    AutoModelWithLMHead
+)

```

Comment:
instead of adding these functions to asset_publish.py directly, I plan to have a separate class for these utilities and call that function from asset_publish.py. Is that fine?